Mathlib/SetTheory/ZFC/Rank.lean

```diff
@@ -201,6 +201,15 @@ theorem rank_range {α : Type*} [Small.{u} α] (f : α → ZFSet.{u}) :
   · simpa [rank_le_iff, ← succ_le_iff] using Ordinal.le_iSup _
   · simp [rank_lt_of_mem]
 
+@[simp]
+theorem rank_iUnion {α : Type*} [Small.{u} α] (f : α → ZFSet.{u}) :
```

Comment:
I think this should go right before `rank_union`, since we can then golf it by using this.